document/FileFormat.mdk

```diff
@@ -913,16 +941,28 @@ An `invocation` object may contain a property named `commandLine` whose value is
 containing the runtime parameters with which the tool was invoked.
```

Comment:
Ok, let's go with that it "shall" include the exe/script name, but with the caveat that path information can be redacted (as it is in the example).

---
In reply to: [75181386](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75181386) [](ancestors = 75181386)